c7n/resources/dms.py

```diff
@@ -45,6 +52,13 @@ def get_source(self, source_type):
     def get_arns(self, resources):
         return [r['ReplicationInstanceArn'] for r in resources]
 
+    def get_tags(self, resources):
+        client = local_session(self.session_factory).client('dms')
+        for r in resources:
+            r['Tags'] = client.list_tags_for_resource(
```

Comment:
this could use a retry on ThrottlingException, per http://docs.aws.amazon.com/dms/latest/APIReference/CommonErrors.html